plugins/rts54hub/README.md

```diff
@@ -0,0 +1,11 @@
+Realtek RTS54 HUB Support
+=========================
+
+Introduction
+------------
+
+This plugin allows the user to update any supported hub and attached downstream
+ICs using a custom HUB-based flashing protocol.
+
+Other devices connected to the RTS54xx using I2C will be supported soon.
```

Comment:
I think it would be good to disambiguate that HUB protocol and endpoints are different than HID protocol and endpoints.